app/lib/methods/search.ts

```diff
@@ -15,10 +15,17 @@ export const localSearchSubscription = async ({ text = '', filterUsers = true, f
 	const searchText = text.trim();
 	const db = database.active;
 	const likeString = sanitizeLikeString(searchText);
+	const slugifiedString = slugifyLikeString(searchText);
 	let subscriptions = await db
 		.get('subscriptions')
 		.query(
-			Q.or(Q.where('name', Q.like(`%${likeString}%`)), Q.where('fname', Q.like(`%${likeString}%`))),
+			Q.or(
+				Q.where('sanitized_fname', Q.like(`%${slugifiedString}%`)),
+				// TODO: We still need the others comparisons, otherwise we needed to clear the cache
```

Comment:
This comment doesn't quite document what's happening nor creates an action for the future.
Can you elaborate?